cpp/src/plasma/malloc.cc

```diff
@@ -40,7 +42,7 @@ int fake_munmap(void*, int64_t);
 #define USE_DL_PREFIX
 #define HAVE_MORECORE 0
 #define DEFAULT_MMAP_THRESHOLD MAX_SIZE_T
-#define DEFAULT_GRANULARITY ((size_t)128U * 1024U)
+#define DEFAULT_GRANULARITY ((size_t) 1024U * 1024U * 1024U) // 1GB
```

Comment:
As part of a separate PR we should implement reserving and mmapping as much of the configured memory capacity as possible on startup.  This would alleviate the need for `DEFAULT_GRANULARITY` altogether.